dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/managed/WCManagedObjectImpl.java

```diff
@@ -13,6 +13,11 @@
 import com.ibm.websphere.ras.annotation.Sensitive;
 import com.ibm.ws.managedobject.ManagedObject;
 import com.ibm.ws.managedobject.ManagedObjectContext;
+import com.ibm.ws.managedobject.ManagedObjectException;
```

Comment:
Could be that this whole class is redundant now ... there should always be a "normal" managed object, this one shouldn't be needed.

Reply:
In my professional opinion, lets not go around deleting entire classes until after we've released